### ISSUE TITLE
[BUG] Muliple Proton emails in Eppie

### ISSUE THREAD

**Issue by i2** [Eppie-io/Eppie-App#471]
### What Would You Like to Be Added?

Hi everyone,

I added my first Proton email without any issue. When trying to add my second Proton email in Eppie (correct username, password and 2FA), it does not work... as in I press on Apply button, it plays the dots (processing) but then it doesn't proceed. Nothing happens.

It would be nice to be able to add multiple Proton emails...
and as a matter of fact multiple of any email provider (Gmail etc.) <- for the latter, I have not tested so not sure if it already works!

Thank you!

Version: v1.0.118.0-preview (latest as of today)

### Why is This Needed?

I believe this is a typical feature of any email client

**Comment by fokuspokus** [comment]
Hi @i2!
 
Thank you for your report!
 
Eppie does support multiple email accounts, and during internal testing, we could not reproduce the issue. However, we want to find out what it is and help resolve this as quickly as possible.

Could you please try the following:
 
Reorder Account Setup:
Remove both accounts (including the first ProtonMail account).
Try adding the second ProtonMail account first and see if it works.
If successful, add the first ProtonMail account afterwards.
 
Once you’ve tried these steps, let us know the outcome. This will help us pinpoint whether the issue lies in account-specific settings or elsewhere.
 
Thank you for helping us improve Eppie!

**Comment by i2** [comment]
Thanks for your reply @fokuspokus

I removed my 1st proton account (well the only one I could add in Eppie, really). [Off-topic: even after removing the account, contacts still remain in the app. No "Select all" option for deleting them and doing so one by one is tedious. Just a note.]

Closed and reopened Eppie. Tried to add my other proton account, but still same issue. The dots dance on the screen and then nothing!

I tried on a third account, and still same issue. I confirm I can log in normally on my browser.

P.S. I have a hunch that the issue might be because of captcha checking. When I login on my browser, it ask to solve a captcha.

Hope this helps trying to figure out where the issue lies.

**Comment by fokuspokus** [comment]
@i2,
You are right. Eppie cannot display captcha yet. This is probably the root of the problem. While we are on it, you could try to solve captcha in the browser to get rid of it, then attempt to log in with Eppie again.
